.github/workflows/update-wiki.sh

```diff
@@ -0,0 +1,48 @@
+conversion() {
+
+    readarray -t categories < <(cat $GITHUB_WORKSPACE/etc/categories | sed 's/$/|/g' | awk -F'|' '{print $2}' | sort - | uniq)
+    readarray -t script_name < <(cat $GITHUB_WORKSPACE/etc/categories | sed 's/$/|/g' | awk -F'|' '{print $1}')
+    readarray -t script_category < <(cat $GITHUB_WORKSPACE/etc/categories | sed 's/$/|/g' | awk -F'|' '{print $2}')
+
+    for category in "${categories[@]}"; do
+        if [[ "$category" == "hidden" ]]; then
+            continue
+        fi
+        echo "# $category"
+        iter=0
+        for script in "${script_name[@]}"; do
+            if [[ "${script_category[$iter]}" == "$category" ]] && [[ -a "$GITHUB_WORKSPACE/apps/$script" ]]; then
+
+                # determine if app is arm64, arm32, or both
+                if [[ -a "$GITHUB_WORKSPACE/apps/$script/install" ]]; then
+                    arch="ARM32/ARM64"
+                elif [[ -a "$GITHUB_WORKSPACE/apps/$script/install-32" ]]; then
+                    if [[ -a "$GITHUB_WORKSPACE/apps/$script/install-64" ]]; then
+                        arch="ARM32/ARM64"
+                    else
+                        arch="ARM32 ONLY"
+                    fi
+                elif [[ -a "$GITHUB_WORKSPACE/apps/$script/install-64" ]]; then
+                    arch="ARM64 ONLY"
+                else
+                    arch="Package app"
+                fi
+                script_website=$(cat "$GITHUB_WORKSPACE/apps/$script/website" 2>/dev/null)
+                script_credits=$(cat "$GITHUB_WORKSPACE/apps/$script/credits" 2>/dev/null)
+                echo ""
+                script_url=$(echo $script | sed -e 's/ /%20/g')
+                echo "### ![This is an image](https://github.com/Botspot/pi-apps/blob/master/apps/$script_url/icon-24.png) ***[$script](https://github.com/Botspot/pi-apps/tree/master/apps/$script_url)***"
```

Comment:
Maybe use the larger icon? The smallrr one looks a bit bad in my opinion.

Reply:
<img width="686" alt="image" src="https://user-images.githubusercontent.com/28281419/145264656-eae96968-4a78-465f-bb18-53ad22ef1834.png">
I feel like the larger one is just too big... let me see if it can be scaled or something

Reply:
### <img src="https://github.com/Botspot/pi-apps/blob/master/apps/Color%20Emoji%20font/icon-64.png" height=32> ***[Color Emoji font]

yup I can scale it down and use the larger one

Reply:
better now? before it was the 24x24 icon, now its the 64x64 icon scaled down by half to 32x32

Reply:
Much Better!